utils.js

```diff
@@ -10,7 +10,7 @@ async function getSelfUser(endpoint = '/users/self') {
     if (endpoint === '/users/self') {
       const self_user = await res.json();
       if (res.status === 200) {
-        return self_user;
+        return self_user?.user || self_user;
```

Comment:
Why `self_user?.user` ?

Reply:
Because while running the test cases user details were present under the user key. On the other hand while running it locally, user details can be found inside self_user.

Reply:
[Here is the](https://github.com/Real-Dev-Squad/website-dashboard/pull/606/files#diff-b6afa233efae4f2d689c44e64e59a981f7f2fe79f45c971f29a5040bfe677823L60) previous implementation, in which they were manually triggering the functions.